dashboard/src/t5gweb/templates/ui/engineer.html

```diff
@@ -0,0 +1,322 @@
+{% extends "skeleton.html" %}
```

Comment:
on a similar note, this duplicates a lot of `account.html`. you could add some conditionals to change the header based on whether `engineer` or `account` was supplied, and only show the jira/bug table if its by account. maybe rename account.html to something more generic like `details.html` or something like that?

Reply:
agree, merged engineer.html template into account.html, no need for a new one, if needed in the future we can extend it and rename it to details.html